scripts/exgfs_atmos_gempak_meta.sh

```diff
@@ -97,11 +126,12 @@ do
   $APRUNCFP $DATA/poescript
   export err=$?; err_chk
 
-  if [ $fhr -eq 126 ] ; then
-    fhr=`expr $fhr + 6`
-  else
-    fhr=`expr $fhr + $fhinc`
-  fi
+      typeset -Z3 fhr
+      if [ $fhr -eq 126 ] ; then
+        let fhr=fhr+6
```

Comment:
Use arithmetic expansion instead: `(( fhr = 10#fhr + 6 ))`
(The 10# is necessary because otherwise zero-padded numbers are treated as octals)

Reply:
Will update in `develop` branch.